docs/slack.rst

```diff
@@ -168,6 +168,12 @@ If you still can't find a suitable channel, you can propose a new channel. To pr
 3. Post in **#meta** requesting a channel. Be sure to share your proposed channel name, description, and to tag your cosponsors.
 
 The Write the Docs moderation team will review your proposal.
+If your proposal is approved, then you and your consponsors must:
+
+#. Seed discussion in the new channel.
+   For example, ask an open-ended question, share a link to discuss, or post a hot take.
+#. Make an announcement in **#general** and channels related to the new channel's topic.
+   For example, if there's a new a channel for underwater basket weaving, then make an announcement like "Join us in #underwater-basket-weaving to discuss techniques, materials, and what to store in all these baskets."
 
```

Comment:
Do we add down here something to the effect of "Channels which remain largely empty or quiet may be closed by the Moderation team."

Reply:
I don't think we should make a threat we're probably never going to act on. I've never got myself together enough to do some relatively easy channel renaming. I just don't the mod team actively _looking_ for empty channels to kill.